libs/glib2/patches/010-pcre.patch

```diff
@@ -1,10 +1,10 @@
 --- a/glib/meson.build
 +++ b/glib/meson.build
-@@ -366,6 +366,7 @@ pcre2_static_args = []
+@@ -404,6 +404,7 @@ pcre2_static_args = []
  
  if use_pcre2_static_flag
    pcre2_static_args = ['-DPCRE2_STATIC']
 +  pcre2 = pcre2.as_link_whole()
  endif
  
- glib_c_args = ['-DG_LOG_DOMAIN="GLib"', '-DGLIB_COMPILATION'] + pcre2_static_args + glib_hidden_visibility_args
+ glib_c_args = ['-DG_LOG_DOMAIN="GLib"'] + glib_c_args_internal + pcre2_static_args
```

Comment:
huh. I wonder if this fixes the host static pcre mess.

Reply:
I'll look at this as soon as this version bump is upstream.

Reply:
How can I reproduce the issue with the static pcre2? At least for the ARMv7 target, I can remove the patch and I have no build breakages.

Reply:
Remove -Dforce_fallback_for=libpcre2-8

then

```
make package/glib2/host/{clean,compile}
make package/vala/host/{clean,compile}
```

Reply:
I did as you suggested but vala compiles without problems.

Reply:
I have updated pcre2 to 10.44. glib2 could successfully detect the new version and vala was also built without any issues.